go.mod

```diff
@@ -220,3 +220,5 @@ require (
 	gopkg.in/ini.v1 v1.66.4 // indirect
 	gopkg.in/square/go-jose.v2 v2.5.1 // indirect
 )
+
+replace github.com/minio/console => github.com/minio/console v0.19.4-0.20220728202016-3b11556f4b9d
```

Comment:
Don't need to use replace tag @donatello all you need to do is to update `go.mod` by doing 

```
go get -u github.com/minio/console@master
```